types/parse-prefer-header/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for node-persist 1.0.0
```

Comment:
You would have to fix the version to be `major.minor` only to pass the CI test.
You could run same test locally calling:
```bash
npm i
```
and:
```bash
npm run lint parse-prefer-header
```
```bash
npm test
```